core/templates/dev/head/profile/profile.html

```diff
@@ -40,6 +40,12 @@ <h1 class="oppia-profile-username"><strong>{{PROFILE_USERNAME}}</strong></h1>
                 <em>This user has not supplied a bio yet.</em>
               </span>
             </p>
+            <p>
+              <span ng-if="firstContributionDatetime">
+                Contributing since
+                <[getLocaleDateString(firstContributionDatetime)]></span>
```

Comment:
Got it. I'm curious though...why is it not needed? I always thought the tags needed to be in a pair, like <span> with </span>